impl/sql/mutations/mutations_test.go

```diff
@@ -35,110 +37,210 @@ func newDB(t testing.TB) *sql.DB {
 	return db
 }
 
-func fillDB(t *testing.T, ctx context.Context, m mutator.Mutation, factory *testutil.FakeFactory) {
+func fillDB(ctx context.Context, t *testing.T, m mutator.Mutation, factory *testutil.FakeFactory) {
 	for _, mtn := range []struct {
-		epoch    int64
-		index    []byte
-		mutation []byte
+		mutation    *tpb.SignedKV
+		outSequence uint64
 	}{
-		{1, []byte("index1"), []byte("mutation1")},
-		{2, []byte("index2"), []byte("mutation2")},
+		{
+			&tpb.SignedKV{
+				KeyValue: &tpb.KeyValue{
```

Comment:
nit: to save screen space, and slightly increase readability, each `KeyValue` could all be on one line.